File: QE-SELENIUM-PYTHON/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    driver.get("https://training-support.net/webelements/target-practice")

    print(driver.title)

    print(driver.find_element(By.XPATH,"//h3[@class='text-3xl font-bold text-orange-600']").text)

    print(driver.find_element(By.XPATH,"//h5[@class='text-3xl font-bold text-purple-600']").value_of_css_property("color"))

    print(driver.find_element(By.XPATH,"//button[contains(text(),'Purple')]").get_attribute("class"))

    print(driver.find_element(By.XPATH,"//button[contains(text(),'Slate')]").text)
    
    driver.quit() 

File: QE-SELENIUM-JAVA/demo/src/main/java/demos/Activity4.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity4 {
    public static void main(String[] args) {
        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/target-practice");
        System.out.println(driver.getTitle());
        WebElement header3 = driver.findElement(By.xpath("//h3[@class='text-3xl font-bold text-orange-600']"));
        System.out.println(header3.getText());
        WebElement header5 = driver.findElement(By.xpath("//h5[@class='text-3xl font-bold text-purple-600']"));
        System.out.println(header5.getCssValue("color"));
        WebElement purpleButton = driver.findElement(By.xpath("//button[contains(text(),'Purple')]"));
        System.out.println(purpleButton.getAttribute("class"));
        WebElement slateButton = driver.findElement(By.xpath("//button[contains(text(),'Slate')]"));
        System.out.println(slateButton.getText());
        driver.quit();
    }
}